classes/controllers/modals/editorDecision/PKPEditorDecisionHandler.inc.php

```diff
@@ -32,7 +32,21 @@ function authorize($request, &$args, $roleAssignments) {
 		import('lib.pkp.classes.security.authorization.internal.ReviewRoundRequiredPolicy');
 		$this->addPolicy(new ReviewRoundRequiredPolicy($request, $args, 'reviewRoundId', $reviewRoundOps));
 
-		return parent::authorize($request, $args, $roleAssignments);
+		$success = parent::authorize($request, $args, $roleAssignments);
```

Comment:
Just a suggestion to take or leave:

    if (!parent::authorize($request, $args, $roleAssignments)) return false;

No need to worry about `$success` later on, and no sense running later validation if we already know it's failed.

Reply:
:+1: